bodhi/server/mail.py

```diff
@@ -339,6 +340,34 @@
 """
 
 
+def read_template(name):
+    """
+    Read template text from file.
+
+    Args:
+        name (basestring): The name of the email template stored in 'release' table in database.
+    Returns:
+        basestring: The text read from the file.
+    """
+    location = config.get('mail.templates_basepath')
+    module, final = location.split(':')
```

Comment:
Since the following code is reused from errors.py, we could make it a function and call this function in errors.py, here and in the tests.

The function could be defined in util.py for example